FR #1124: Add Tag Scheme to glossary (#2112)
FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec: specification/glossary.md

A metadata label assigned to a resource to provide information about it or to categorize it for organizational and management purposes.

<a name="glossary:tag-scheme"><b>Tag Scheme</b></a>

> A distinct framework for assigning metadata to charges, resources, or other assets. Different metadata types within a single provider (e.g., "tags" vs. "labels") are considered separate schemes. They can be user-defined (custom input) or provider-defined (system-controlled).

Author: Thanks for the suggestion Shawn. I accepted because it's an improvement over what it was, but I'm not sure on the last sentence.

```suggestion
A distinct framework for assigning metadata to charges, resources, or other assets. Different metadata types within a single provider (e.g., "tags" vs. "labels") are considered separate schemes. They can be user-defined (allowing user input) or provider-defined (controlled by the provider).
```

Reviewer: Understood.  How about:

```suggestion
A distinct framework for assigning metadata to charges, resources, or other assets. Different metadata types within a single provider (e.g., "tags" vs. "labels") are considered separate schemes. They can be user-defined (i.e., allowing user input) or provider-defined (i.e., controlled by the service provider).
```

Reviewer 2: I have to double down here and agree with Shawn on this clarification.

> A distinct framework used to apply metadata to resources, other types of assets, or directly to charges. E.g., if a provider supports labels and tags or multiple different types of tags, each would be considered its own Tag Scheme. Tag Schemes that take input from the user are referred to as user-defined tag schemes and those that are completely controlled by a provider or data generator are referred to as provider-defined or data generator-defined tag schemes, respectively.

Reviewer: Wordsmithing for clarity and brevity.  Feel free to pick the pieces you like.

```suggestion
A distinct framework for assigning metadata to charges, resources, or other assets. Different metadata types within a single provider (e.g., "tags" vs. "labels") are considered separate schemes. They can be user-defined (custom input) or provider-defined (system-controlled).
```
